graphql-database-manager-core/src/main/java/com/fleetpin/graphql/database/manager/Query.java

```diff
@@ -0,0 +1,66 @@
+package com.fleetpin.graphql.database.manager;
+
+import java.util.Objects;
+
+public class Query<T extends Table> {
+
+	private final Class<T> type;
+	private final String startsWith;
+	private final String after;
+	private final String until;

```

Comment:
I think we should remove until since we did not implemented it.
Being a builder is easy to add in the future if we want it